models/comments.model.js

```diff
@@ -38,7 +39,30 @@ function insertComment(body, commentToBeInserted, article_id, username) {
       return res.rows[0].body;
     });
 }
+
+function deleteCommentFromDb(comment_id) {
+  if (isNaN(comment_id)) {
+    return Promise.reject({ status: 400, message: "bad request" });
+  } else {
```

Comment:
Does PSQL error throw an error we could use for this?

Reply:
Before sending the query to db and wait for psql return the error, I thought I could handle it with this if statement. Please let me know if this is a bad practice. I added your recommended test. All good